.travis.yml

```diff
@@ -64,6 +64,7 @@ before_install:
 install:
   - "travis_retry pip install -r .travis-${REQUIREMENTS}-requirements.txt"
   - "travis_retry pip install -e .[all]"
+  - "./examples/app-recreate.sh"
```

Comment:
For sure if any part fails than the tests are not passing and you now that the example should be fixed.
I put there because the problem is that is difficult to predict how many time it requires and many times tests are failing only because it was slow.